### ISSUE TITLE
support setting the socket.io path

### ISSUE THREAD

**Issue by matthewp** [donejs/done-serve#7]
Perhaps with a `--socketio-path` option.

**Comment by matthewp** [comment]
By default socket.io requests are routed through `/socket.io`.  This is configurable when setting up a socket.io server, however.  This change would add a `--socketio-path` option so that you could use a different route for the socket.io requests.

**Comment by matthewp** [comment]
@marshallswain I can't remember, are you the person who requested this feature?

**Comment by marshallswain** [comment]
I don't think so.  This doesn't trigger anything in my memory, at least, but that could be due to old age.
